### PR TITLE
Load storage account secrets from Key vault

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1248

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -7,13 +7,32 @@ properties([
   pipelineTriggers([cron('*/30 * * * *')])
 ])
 
+def combinedSecrets = [
+  'reform-scan-aat': [
+    secret('storage-account-secondary-key', 'TEST_STORAGE_ACCOUNT_KEY'),
```
Comment (doncem) [inline Jenkinsfile_nightly:12]: :heart:

Comment (nitinprabhu) [inline Jenkinsfile_nightly:12]: why do we need the storage keys ? We can retrieve SAS token using endpoint `/token/sscs` and upload to blob storage  ?

Comment (nitinprabhu) [reply Jenkinsfile_nightly:12]: Advantage is no keys needs to be loaded less config :) plus you don't have master access

Comment (aliveni) [reply Jenkinsfile_nightly:12]: Ok, I'll raise new PR

**Comment by aliveni** [comment]
No secrets will be used if we use a SAS token to upload the zip files.

**Pull request closed without merging**